README.md

```diff
@@ -262,6 +262,16 @@ To check whether dnsmasq name server at 127.0.0.1 can resolve subdomains of dev.
 app.dev.gov.uk.		0	IN	A	127.0.0.1
 ```
 
+### How to: work with local gems
+
+Provide a local gem path relative to the location of the Gemfile you're editing:
+
+```ruby
+gem 'govuk_publishing_components', path: '../govuk_publishing_components' 
+```
+
+You can reference any applications you have under `~/govuk`, which is mounted as a volume to your container at `/govuk`.
```

Comment:
I'm not understanding this - do we need to say it?

Reply:
Probably not.